src/OSPSuite.InstallationValidator.Core/Domain/StartableProcess.cs

```diff
@@ -46,14 +46,15 @@ public void Dispose()
 
       public virtual void Wait(CancellationToken token)
```

Comment:
We don't have such a case now, but I don't see why we should make it a requirement to supply a cancellation token where you are not going to use it (in case you are not waiting for the result)